Initial update to ESR

woodbe opened the pull request:
This is the updated ESR with proposed changes for discussion

I have left notes where I haven't completed updates (or know that something may need to be done there)

Commit:
Initial update to ESR

This is the updated ESR with proposed changes for discussion

n-kai reviewed (approved):
I only reviewed the first section (Status) and my comments are summarized in Issue #17. I will create the PR for the other sections later.

Commit:
Editorial changes

This is to close #17

I also found some misspellings.

Graphic was updated to add another alternative line from the biometrics to the CMFA PAD

Commit:
updates from call

Updates based on the 1/14 call
